src/bin/conserve.rs

```diff
@@ -104,6 +106,12 @@ fn make_clap<'a, 'b>() -> clap::App<'a, 'b> {
             .multiple(true)
             .global(true)
             .help("Be more verbose (log all file names)"))
+        .arg(Arg::with_name("exclude")
+            .long("exclude")
+            .short("e")
+            .global(true)
```

Comment:
This actually shouldn't be global: it will apply to many commands but not all, for example not `init`.